packages/api-catalog-spa/src/layouts/AppLayout/AppLayout.tsx

```diff
@@ -0,0 +1,29 @@
+import '@one-platform/opc-footer/dist/opc-footer';
+import { FC } from 'react';
+import { Flex, FlexItem, Page } from '@patternfly/react-core';
+import { Container } from 'components/Container';
+interface Props {
```

Comment:
As this interface is empty it can be removed..

Reply:
Updated